<launch>
  <node pkg="hebi_cpp_api_examples" type="omni_base_node" name="omni_base_node" />
</launch>
